nexus/src/internal_api/http_entrypoints.rs

```diff
@@ -1476,4 +1477,37 @@ impl NexusInternalApi for NexusInternalApiImpl {
             .instrument_dropshot_handler(&rqctx, handler)
             .await
     }
+
+    async fn quiesce_start(
+        rqctx: RequestContext<Self::Context>,
+    ) -> Result<HttpResponseUpdatedNoContent, HttpError> {
```

Comment:
Should we be concerned about the case where:

- Nexus is quiescing (old Nexuses are running, as are new Nexuses)
- The old nexuses all quiesce, but one of them, for whatever reason, thinks that quiescing isn't done. Maybe it can't hear back from other old nexuses for some reason.
- The new nexuses boot, perform schema changes, resume their lives
- The old nexus retries: it reads internal DNS, sees records for all the new nexuses, and tells them to quiesce
- Now the system has an outage

Should we have an argument here, that's "only start quiescing if you are ________", where "_______" might be "running an old version", or something like that? Just something to make it conditional?